lib/src/audit.dart

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+import 'dart:async';
+
+/// Creates a StreamTransformer which only emits once per [duration],
```

Comment:
Ah cool. I think it could be helpful to call out the differences from debounce and throttle in the doc comment.